package.json

```diff
@@ -6,22 +6,30 @@
     "scripts": {
         "build": "tsc",
         "test": "jest",
-        "start": "node ./dist/index"
+        "start": "node ./dist/index",
+        "dev": "ts-node-dev --respawn src/index.ts",
+        "format": "prettier ./ -w"
     },
     "keywords": [],
     "author": "",
     "license": "ISC",
     "devDependencies": {
+        "@types/config": "0.0.41",
         "@types/express": "^4.17.13",
         "@types/jest": "^27.4.1",
         "@types/node": "^17.0.23",
+        "@types/supertest": "^2.0.12",
         "jest": "^27.5.1",
+        "prettier": "^2.6.1",
+        "supertest": "^6.2.2",
         "ts-jest": "^27.1.4",
+        "ts-node": "^10.7.0",
+        "ts-node-dev": "^1.1.8",
         "typescript": "^4.6.3"
     },
     "dependencies": {
-        "@types/supertest": "^2.0.12",
-        "express": "^4.17.3",
-        "supertest": "^6.2.2"
```

Comment:
I've moved this and the types up to devDependencies because our build won't be dependent on them